lib/challenge.js

```diff
@@ -60,6 +92,19 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function pet(name, species, breed, noise) = {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+}
+
+var Pet = new getPetInfo();
```

Comment:
This statement invokes a nonexistent function `getPetInfo` as a constructor (the `new` keyword is used only with constructor functions) and assigns the result to `Pet`, causing your code to lose the reference to your constructor function above.